test/upgrade/simpleResources.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: simple-task
+  namespace: upgrade
+spec:
+  steps:
+  - name: step
+    image: gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
+    command: ['/bin/bash']
+    workingDir: /dir
+  stepTemplate:
+    image: gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
+    command: ['/bin/bash']
+  sidecars:
+  - name: server
+    image: alpine/git:v2.26.2
+    command: ['/bin/bash']
+    workingDir: /dir
+    script: echo test
+  volumes:
+    - name: messages
+      emptyDir: {}
+  params:
+  - name: PARAM
+    description: param des
+    type: string
+    default: "1"
+  workspaces:
+  - name: workspace
+    description: description
+    mountPath: /foo
+    readOnly: true
+    optional: true
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: simple-pipeline
+  namespace: upgrade
+spec:
+  description: foo
+  tasks:
+  - name: generate-result
+    params:
+    - name: STRING_LENGTH
+      value: 1
+    taskSpec:
+      params:
+        - name: task1-result
+          type: string
+          default: "1"
+      steps:
+        - image: alpine
+          onError: continue
+          name: exit-with-255
+          script: |
+            exit 255
+    timeout: 60s
```

Comment:
It can often take > 60s to schedule a pod, I'd increase the timeout or omit it to reduce flakes